.circleci/scripts/git-diff-default-branch.ts

```diff
@@ -1,6 +1,6 @@
 import { exec as execCallback } from 'child_process';
-import fs from 'fs';
-import path from 'path';
+import * as fs from 'fs';
```

Comment:
What was the reason for these import changes?

Reply:
eslint: Module '"fs"' has no default export.

Reply:
How did it run before then 🤔

Reply:
My hunch is that `yarn lint` isn't running on these files? :-/